content/en/events/2021/kcsna/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Contributor Summit North America 2021"
+type: docs
+aliases: [ "/kcsna2021" ]
+weight: 1
+description: >
+  The Kubernetes Contributor Summits are returning for 2021!
+
+---
+
+<img align="center" alt="Kubernetes Contributor Summit in Los Angeles, California" style="max-width:80%;" src="/events/past-events/2021/kcsna/banner.png">
```

Comment:
All file paths needs to be updated
Find -> Replace
`/events/past-events/2021/kcsna` -> `/events/2021/kcsna`